make_queue/templates/make_queue/machine_card.html

```diff
@@ -5,7 +5,7 @@
 <div class="ui {% if color_bottom %}{% card_color_from_machine_status machine %} {% endif %}card">
     <link rel="stylesheet" href="{% static 'make_queue/css/stream.css' %}">
     {% if machine.machine_type.has_stream %}
-        <img class="stream image" name="{% sanitize_stream_name machine %}"
+        <img class="stream image" data-stream="{{ machine.stream_name }}" name="{% sanitize_stream_name machine %}"
```

Comment:
Is the `name` attribute used anymore? If not can it and the `sanitize_stream_name` tag be removed?

Reply:
It is removed